test/functional/services/remote/remote.ts

```diff
@@ -38,6 +38,17 @@ export async function RemoteProvider({ getService }: FtrProviderContext) {
   const coveragePrefix = 'coveragejson:';
   const coverageDir = resolve(__dirname, '../../../../target/kibana-coverage/functional');
   let logSubscription: undefined | Rx.Subscription;
+  type BrowserStorage = 'sessionStorage' | 'localStorage';
+
+  const clearBrowserStorage = async (storageType: BrowserStorage) => {
+    try {
+      await driver.executeScript(`window.${storageType}.clear();`);
+    } catch (error) {
+      if (!error.message.includes(`Failed to read the ${storageType} property from 'Window'`)) {
```

Comment:
This used to have single quotes around the storage type, which one is correct?

Reply:
You are right Spencer, I actually tested with quotes 😫